app/javascript/token.js

```diff
@@ -33,7 +38,7 @@ document.addEventListener('turbolinks:load', () => {
   const iframe = document.createElement('iframe')
   const params = {
     response_type: 'token',
-    scope: 'upload_run',
+    scope: 'upload_run+websocket_sign_in',
```

Comment:
We talked about this offline but as a reminder -- let's change this to be a scope for `conduct_race` or similar.